flags/mod.ts

```diff
@@ -52,6 +52,9 @@ export interface ParseOptions {
   /** A string or array of strings argument names to always treat as strings. */
   string?: string | string[];
 
+  /** A string or array of strings argument names to always treat as arrays. */
+  collect?: boolean | string | string[];
```

Comment:
I guess nobody would use `collect: true` as most cli tools tend to have `--help` or `--version` and they shouldn't be collectable. Do you think of any use case of `collect: true`?

Reply:
yeah maybe it's not needed. i can remove it.